example/ellip_l9.inp

```diff
@@ -10,7 +10,7 @@
 14   47.0
 2.23
 1.0       30.0        0.1
-5000
+50
```

Comment:
ik vermoed dat je deze aanpassing maakt om het sneller te doen lopen? Want 50 fotonen simuleren is uiteraard niet echt representatief.

Reply:
Oeps, deze had er niet mogen bijkomen. Heb dat gedaan om valgrind wat sneller te doen lopen. Zal even snel commit aanpassen.